contrib/natss/pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -67,7 +68,12 @@ func createReceiverFunction(s *SubscriptionsSupervisor, logger *zap.SugaredLogge
 		logger.Infof("Received message from %q channel", channel.String())
 		// publish to Natss
 		ch := getSubject(channel)
-		if err := stanutil.Publish(s.natssConn, ch, &m.Payload, logger); err != nil {
+		message, err := json.Marshal(m)
```

Comment:
I don't think anything else directly marshals this struct, please add the `json` tags to it (similar to the API types). Also put a comment there, saying that it does get marshaled and therefore needs to be backwards compatible for at least one full release.

Reply:
done